src/main/resources/application-local.yml

```diff
@@ -14,8 +11,34 @@ spring:
         show-sql: true
         format_sql: true
 
+  flyway:
+    enabled: false
+    placeholders:
+      email: ${FLYWAY_EMAIL}
+      password: ${FLYWAY_PASSWAORD}
+      role: ${FLYWAY_ROLE}
+      enabled: ${FLYWAY_ENABLED}
```

Comment:
i suppose placeholders should be moved to the `application.yml` coz we have flyway turned of in local
and want to turn it on for deployment environments, like Render, AWS, etc...., right ?